modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/ImageMosaicConfigHandler.java

```diff
@@ -1379,6 +1379,17 @@ public void updateConfiguration(GridCoverage2DReader coverageReader,
                 configBuilder.setCrs(actualCRS);
             }
 
+            String noData = IndexerUtils.getParameter(Prop.NO_DATA, indexer);
+            if(noData != null && !noData.isEmpty()) {
+                try {
+                    double noDataValue = Double.parseDouble(noData);
+                    configBuilder.setNoData(noDataValue);
+                } catch (NumberFormatException e) {
+                    throw new RuntimeException("Invalid NoData specification " + noData + ", was expecting a number", e);
```

Comment:
Question on my side:
I see in the Utils class that a FINE log level will be raised reporting an unparseable nodata in case a NumberFormatException occurs. 
Should we adopt a similar approach here (maybe WARNING) or do you think it is better to stop the mosaic configuration at all, instead?

Reply:
Personally I have a preference for fail fast, at least against configuration, otherwise one just looks around without understanding. Maybe doing both (log and exception) is better